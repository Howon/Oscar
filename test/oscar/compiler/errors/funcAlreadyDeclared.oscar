def add5(int: x) => int = {
  return (x + 5);
}

def add5(int: x) => int = {
  return (x + 5);
}

def main() => unit = {
  add5(1) 
}