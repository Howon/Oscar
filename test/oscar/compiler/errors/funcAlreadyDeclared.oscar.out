Error: Value add5 declared already
